src/Files.App/Constants.cs

```diff
@@ -51,7 +51,25 @@ public static class ImageRes
             public const int Folder = 3;
         }
 
-        public static class Shell32
+		public static class FluentIconsPaths
```

Comment:
Is spacing set to tabs?

Reply:
It seems so. I'm wondering if the source code is not using tabs in some places and spaces in others.

Fixed.